invenio_app_ils/ui/src/invenio_app_ils/common/api/location.js

```diff
@@ -0,0 +1,25 @@
+import { http } from './base';
+
+const locationURL = '/locations/';
```

Comment:
Since they are different entities I would separate them under different files, `api/location.js` and `api/internal-location.js`

Reply:
Yes but is really interconnected...